python/lsst/daf/persistence/__init__.py

```diff
@@ -22,16 +22,20 @@
 
 """Python interface to lsst::daf::persistence classes
 """
+from utils import *
```

Comment:
I think these should all be relative (`from .utils import *`) imports, if they refer to the same package.  That may not matter now, but I think it's the convention we use in most other packages and it may be important for Python 3.